docs/reference-guides/block-api/block-templates.md

```diff
@@ -120,6 +120,26 @@ _Options:_
 
 Lock settings can be inherited by InnerBlocks. If `templateLock` is not set in an InnerBlocks area, the locking of the parent InnerBlocks area is used. If the block is a top level block, the locking configuration of the current post type is used.
 
+## Individual block locking
+
+Alongside template level locking, you can lock individual blocks; you can do this using a `lock` attribute on the attributes level. Block-level lock takes priority over the `templateLock` feature. Currently, you can lock moving and removing blocks.
+
+**Block-level locking is an experimental feature that may be removed or change anytime.**
+```js
+attributes: {
```

Comment:
Cool work.

It looks like the documentation needs to be fixed. It differs from the PR's description and feels broken here:

<img width="835" alt="Screen Shot 2021-10-06 at 21 21 33" src="https://user-images.githubusercontent.com/699132/136269328-4f43ee64-3c9e-4335-9c9c-8bab170fdcbc.png">